src/components/FlyoutMenu/FlyoutMenu.story.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { Button, Tabs, Tab, Dropdown } from 'carbon-components-react';
```

Comment:
Same thing here - we can import the components relative to the project instead of directly from `carbon-components-react`

```suggestion
import Button from '../Button';
import Tab from '../Tab';
import Tabs from '../Tabs';
import Dropdown from '../Dropdown';
```